### ISSUE TITLE
Dragging timelines off the current window.

### ISSUE THREAD

**Issue by Onisan2000** [timeglider/jquery_widget#27]
Hi, 
I've noticed that you can mouse drag all of the timelines outside of the timeline frame and loose them until you re-load.
also when clicking on a timeline the underlaying popup box can also pop up half in and half hidden on the frame.
Gary

**Comment by timeglider** [comment]
Right: there's no limit right now to how you drag the timeline — it can drag everything completely out of the frame.  Still need to work on a UI component that either restricts dragging (ceasing the drag when one is at an outer boundary) or gives one a clue about where the data is, off-frame.

We've fixed the modal pop-up by simply making it show up in the center of the screen, rather than trying a fancy conditional positioning. The problem with that has been that the parent element, extending beyond the visible frame, doesn't trigger jquery ui's position to "flip" or "fit" things, since the modals haven't exceeded the parent elements' boundaries. 

I'll leave this open until we have a better way to keep the data constrained in the frame.
